packages/helm-charts/oracle/CELOBRL.yaml

```diff
@@ -3,27 +3,39 @@ oracle:
   aggregation:
     mid:
       maxExchangeVolumeShare: 1
-      maxPercentageDeviation: 0.025
+      maxPercentageDeviation: 0.05
       maxPercentageBidAskSpread: 0.015
   metrics:
     enabled: true
     prometheusPort: 9090
   apiRequestTimeoutMs: 5000
   circuitBreakerPriceChangeThreshold: 0.25
   priceSources: "[
-      [
-        { exchange: 'BINANCE', symbol: 'CELOBUSD', toInvert: false },
-        { exchange: 'BINANCE', symbol: 'BUSDBRL', toInvert: false }
-      ],
-      [
-        { exchange: 'COINBASE', symbol: 'CELOUSD', toInvert: false},
-        { exchange: 'BITSO', symbol: 'USDBRL', toInvert: false }
-      ],
-      [
-        { exchange: 'COINBASE', symbol: 'CELOBTC', toInvert: false },
-        { exchange: 'NOVADAX', symbol: 'BTCBRL', toInvert: false }
-      ]
-    ]"
+			[
+				{ exchange: 'BINANCE', symbol: 'CELOBUSD', toInvert: false },
+				{ exchange: 'BINANCE', symbol: 'BUSDBRL', toInvert: false }
+			],
+			[
+				{ exchange: 'BINANCE', symbol: 'CELOUSDT', toInvert: false },
+				{ exchange: 'BINANCE', symbol: 'USDTBRL', toInvert: false }
+			],
+			[
+				{ exchange: 'BINANCE', symbol: 'CELOBTC', toInvert: false },
+				{ exchange: 'MERCADO', symbol: 'BTCBRL', toInvert: false }
+			],
+			[
+				{ exchange: 'COINBASE', symbol: 'CELOUSD', toInvert: false},
+				{ exchange: 'BITSO', symbol: 'USDBRL', toInvert: false }
+			],
+			[
+				{ exchange: 'COINBASE', symbol: 'CELOBTC', toInvert: false },
+				{ exchange: 'NOVADAX', symbol: 'BTCBRL', toInvert: false }
```

Comment:
good point, I'll add `BINANCEUS:CELOUSD`. The `BINANCEUS:CELOUSDT` rate is unstable

Reply:
do you think it would be possible to use only South American / European / Asian clusters and not a US one instead? BRL trading seems to be centered around Binance International ...